### ISSUE TITLE
Xlib.h compilation failure

### ISSUE THREAD

**Issue by faveoled** [indoorvivants/sn-bindgen#146]
Running bindgen-sbt-plugin 0.0.14
```
[error] -- [E040] Syntax Error: /home/user/Dev/Scala/scala-native/target/scala-3.2.1/src_managed/main/libx.scala:862:137 
[error] 862 |    def apply(`type` : CInt, serial : CUnsignedLongInt, send_event : CInt, display : Ptr[Display], window : Window, colormap : Colormap, new : CInt, state : CInt)(using Zone): Ptr[XColormapEvent] = 
[error]     |                                                                                                                                         ^^^
[error]     |                               an identifier expected, but 'new' found
[error]     |
[error]     | longer explanation available when compiling with `-explain`
[error] -- [E040] Syntax Error: /home/user/Dev/Scala/scala-native/target/scala-3.2.1/src_managed/main/libx.scala:870:17 
[error] 870 |      (!____ptr).new = new
[error]     |                 ^^^
[error]     |                 an identifier expected, but 'new' found
[error]     |
[error]     | longer explanation available when compiling with `-explain`
[error] -- [E040] Syntax Error: /home/user/Dev/Scala/scala-native/target/scala-3.2.1/src_managed/main/libx.scala:886:10 
[error] 886 |      def new : CInt = struct._7
[error]     |          ^^^
[error]     |          an identifier expected, but 'new' found
[error]     |
[error]     | longer explanation available when compiling with `-explain`
[error] three errors found
[error] (Compile / compileIncremental) Compilation failed
[error] Total time: 17 s, completed Jan 20, 2023, 2:25:42 PM
```

**Comment by keynmol** [comment]
That's pretty embarrassing, of all the [keywords I added](https://github.com/indoorvivants/sn-bindgen/blob/94a9e9fb8ef59fb5f69fcac992dc99ad9eb8eb48/modules/bindgen/src/main/scala/render/utils.scala#L38-L48) I forgot the main one.

Let me know if you would like to attempt to fix it and add a test for it - if not, I will address it in a couple of days.

**Comment by faveoled** [comment]
No, I am not that advanced with Scala

**Comment by keynmol** [comment]
I believe this should've been fixed by #154, please reopen if that's not the case
